src/gui/properties/peerlistwidget.cpp

```diff
@@ -52,463 +52,453 @@
 #include "peerlistwidget.h"
 
 PeerListWidget::PeerListWidget(PropertiesWidget *parent):
-  QTreeView(parent), m_properties(parent), m_displayFlags(false)
+    QTreeView(parent), m_properties(parent), m_displayFlags(false)
```

Comment:
If you follow the coding style, follow this all. See CODING_GUIDELINES.md carefully.